src/main/webapp/WEB-INF/views/cars/car_registration.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add new car</title>
+</head>
+<body>
+<h1>Car registration form</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/add">
+    MODEL <input type="text" name="model"><br>
+    MANUFACTURER ID <input type="text" name="manufacturer_id"><br>

```

Comment:
It would be better to give user possibility to select manufacturer from the list.
```suggestion
    MANUFACTURER ID: <br/>
    <c:forEach items="${manufacturers}" var="manufacturer">
        <input type="radio" id="html" name="manufacturer_id" value="${manufacturer.id}" required>
        <label for="${manufacturer.id}">${manufacturer.name}</label><br>
    </c:forEach>
    <br/>
```